recipes/dask-geomodeling/meta.yaml

```
@@ -0,0 +1,62 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "dask-geomodeling" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: da7f1450ba8ac87f61006b49aba8f341d742ca940fdb4d36ed4b783d694bfc7f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
```

I think this recipe can be ``noarch``
```suggestion
  noarch: python
```

You can restrict the python version on the requirements